GEMINI.md

```
@@ -0,0 +1 @@
+Follow the instructions and guidelines specified in the @./AGENTS.md file for all tasks in this project.
```

markdownlint-cli2 (0.17.2)

First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)